index.js

```diff
@@ -365,6 +365,13 @@ function intentHandler(intent) {
         case 'highlightURL':
             highLight('urlbar');
             break;
+        case 'claimPrize':
+            var sidebarProps = {};
+            sidebarProps.id = "all-aboard-reward";
+            sidebarProps.title = "Prize";
+            sidebarProps.url = "./tmpl/reward.html";
+            showSidebar(sidebarProps);
```

Comment:
Hmmmm, `showSidebar` has all kinds a logic that are aimed at the general 5 content sidebars. Things such as workers listening for `cta_complete`, `intent`(which is fine here), and `show`. It also does things like set the step, update the `distribution.id` and also emits the `tokens` event.

I therefore do not think it is a good idea to re-use it here. It would be preferable to create a `showRewardSidebar` function that is specifically made for this. We can then also use that function when showing this after the reward notification.